src/components/avatar/pages/avatar-interactive.jsx

```diff
@@ -3,12 +3,24 @@ import Avatar, {SIZE} from '../Avatar';
 import DocsActiveBlock from 'components/DocsActiveBlock';
 
 const Avatars = () => {
-  const settings = {
-    size: SIZE,
-    border: Boolean,
-    spaced: Boolean,
-    imgSrc: String
-  };
+  const settings = [
+    {
+      name: 'size',
+      values: SIZE
+    },
+    {
+      name: 'border',
+      values: Boolean
+    },
+    {
+      name: 'spaced',
+      values: Boolean
+    },
+    {
+      name: 'imgSrc',
+      values: String
+    }
+  ];
```

Comment:
I had to change the format to allow more options (e.g. 'required') - I wish we could read that from propTypes :(